aws_lambda_powertools/utilities/parameters/secrets.py

```diff
@@ -224,3 +359,88 @@ def get_secret(
         force_fetch=force_fetch,
         **sdk_options,
     )
+
+
+def set_secret(
+    name: str,
+    value: Union[str, bytes],
+    *,  # force keyword arguments
+    client_request_token: Optional[str] = None,
+    **sdk_options,
+) -> str:
+    """
+    Modify the details of a secret or create a new secret if it doesn't already exist.
+    It includes metadata and the secret value.
```

Comment:
I don't understand this sentence :( Either remove it or rewrite it please. What is "metadata"?

Reply:
Yep! Some leftover, I removed the sentence.